sampleSpecs/test.smithy

```diff
@@ -194,3 +193,49 @@ structure ErrorDetails {
     @required
     location: String
 }
+
+
+@http(method: "PUT", uri: "/json-name-test")
+@httpRequestTests([
+    {
+        id: "JsonNameTest",
+        protocol: simpleRestJson,
+        documentation: "This is a test of the jsonName trait",
+        method: "PUT",
+        uri: "/json-name-test",
+        body:
+        """
+              {
+               "foo": "test"
+                }""",
+        bodyMediaType: "application/json",
+        params: {
+            bar: "test"
+        }
+    }
+])
+@httpResponseTests([
+    {
+        id: "JsonNameTest",
```

Comment:
This reproduction is not helped by the fact that both tests are named the same : it's unclear which error belongs to which test, cause the implementation of compliance-tests might change the ordering accidentally.

Reply:
fixed